planb/cli.py

```diff
@@ -35,12 +36,17 @@ def validate_dc_suffix(ctx, param, value):
     return value
 
 
+def parse_filters(ctx, param, value):
+    return json.loads(value)
```

Comment:
Why this long list of parameters if we use only value?

Reply:
And could we add a validation so that it is ensured the result is a list of objects? (couldn't see this validation elsewhere)